nixos/tests/all-tests.nix

```diff
@@ -62,6 +62,7 @@ in
   cjdns = handleTest ./cjdns.nix {};
   clickhouse = handleTest ./clickhouse.nix {};
   cloud-init = handleTest ./cloud-init.nix {};
+  cntr = handleTest ./cntr.nix {};
```

Comment:
It would make sense to have this in the cntr package referenced as `passthru.tests.cntr`. See ferm package for an example.

Reply:
Thanks, done!